code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -596,6 +620,16 @@ GLOBAL_LIST_INIT(brass_recipes, list (
 	table_type = /obj/structure/table/reinforced/brass
 	dynamic_icon_state = TRUE
 
+/obj/item/stack/tile/brass/examine_more(mob/user)
+	. = ..()
+	. += {"Brass describes a class of alloys made primarily from copper and zinc, with colours ranging from yellowish-white, gold, brown, and reddish. \
```

Comment:
does not follow the same code procedures as your previous `examine_more` procs, do you need to reset `.` to equal `list()` or does that not matter here?

Reply:
Gah, I am silly.